dynawo/sources/Solvers/SolverSIM/DYNSolverSIM.h

```diff
@@ -137,23 +136,6 @@ class SolverSIM : public Solver::Impl {
   void printSolveSpecific(std::stringstream& msg) const;
 
  private:
-  /**
-   * @brief integrate the DAE over an interval and recalculate it in case of a z change
-   * @param tNxt next time step
-   */
-  void solveWithStepRecalculation(double &tNxt);
-
-  /**
-   * @brief integrate the DAE over an interval without recalculting the step in case of a z change
-   * @param tNxt next time step
-   */
-  void solveWithoutStepRecalculation(double &tNxt);
-
-  /**
-   * @brief save the initial values of roots, y, and z before the time step
-   */
-  void saveInitialValues();
```

Comment:
not used anymore?

Reply:
Yes, as roots and z are only calculated if the algebraic problem is solved: there is no need to save their initial values. Continuous variable values are saved by the saveContinuousVariables() method.